map/lanelet2_extension/include/lanelet2_extension/utility/query.hpp

```diff
@@ -45,6 +47,11 @@ namespace lanelet
 {
 namespace utils
 {
+inline double getAngleDifference(const double angle1, const double angle2)
```

Comment:
If you use `tier4_autoware_utils`, adding `[[deprecated]]` attribute would be good because this function can be replaced by it.

You should talk to the author of this package about this.

Reply:
getAngleDifference is only used in getClosestLanelet so it won't misunderstanding much but i will add it.

Reply:
Thank you. But if it's not used by any other applications, I think you can remove the function.

Reply:
@mitsudome-r Is it okay for you to use `tier4_autoware_utils` in `lanelet2_extension`? My concern is that it adds a dependency.

Reply:
@mitsudome-r PR can be reverted since it's small change if dependency will be concern

Reply:
@kenji-miyake 
>I think you can remove the function.

yes but this function is where fault happends so it would be better to test it.

>My concern is that it adds a dependency.

@mitsudome-r  said it's ok to use autoware_utils in lanelet2_extension and dependency won't be something to care.

![image](https://user-images.githubusercontent.com/65527974/150063886-e340366a-482c-4c45-835d-462b52740f19.png)

Reply:
4f68b48